api/src/pcapi/routes/backoffice/templates/components/forms/string_field.html

```diff
@@ -18,6 +19,9 @@
          {% if field.flags.max is not none %}max="{{ field.flags.max }}"{% endif %}
          {% if field.flags.min is not none %}min="{{ field.flags.min }}"{% endif %} />
   {% if not hidden_field %}
+    {% if not (field.type == "PCIntegerField" or field.type == "PCOptIntegerField" or field.type == "PCOptHiddenIntegerField" or field.type == "PCDecimalField") %}
```

Comment:
Plutôt que lister et finir par oublier cela, ne serait-il pas préférable d'ajouter un attribut `can_be_cleared` (ou meilleur nom équivalent) aux classes `PCOptStringField` et `PCOptPasswordField`, ainsi attribué à toutes les classes qui en héritent, pour simplement tester l'attribut ?
```suggestion
    {% if field.can_be cleared %}
```
Ou sinon au travers des `field.flags` ?

Reply:
pour moi c'est plus un attribut du field